ovs/vswitch.go

```diff
@@ -126,6 +126,12 @@ func (v *VSwitchService) SetController(bridge string, address string) error {
 	return err
 }
 
+// GetController gets the controller address for this bridge.
+func (v *VSwitchService) GetController(bridge string) (string, error) {
+	address, err := v.exec("get-controller", bridge)
```

Comment:
Just to be sure, let's add an error check and then trim the output on success:

```go
address, err := v.exec("get-controller", bridge)
if err != nil {
    return nil, err
}

return strings.TrimSpace(string(address)), nil
```

Reply:
Done. Thank you.

Reply:
Done. Thank you.

Reply:
Please add the error check.  In most cases, it isn't safe to do anything with the first return value if error is not nil.

Reply:
Sorry, i Missed that last time.
I added "" instead of nil as its a string and compiler complained.